premailer/premailer.py

```diff
@@ -402,25 +402,23 @@ def transform(self, pretty_print=True, **kwargs):
         # URLs
         #
         if self.base_url:
-            if not self.base_url.endswith('/'):
-                self.base_url += '/'
+            if not urlparse(self.base_url).scheme:
+                raise ValueError('Base URL must start have a protocol')
```

Comment:
Erm, I think the phrasing of the Value Error is still wrong... And @graingert suggested using "scheme" instead of protocol (which would also apply to the name of the test)